coalib/parsing/CliParsing.py

```diff
@@ -87,3 +87,31 @@ def parse_custom_settings(sections,
                                origin=origin,
                                section_name=key_touple[0],
                                from_cli=True)
+
+
+def check_conflicts(cli_sections, log_printer):
```

Comment:
Maybe you could do 2 mutually exclusive groups ?

On Sun, Mar 13, 2016 at 11:15 AM, Adhityaa Chandrasekar <
notifications@github.com> wrote:

> In coalib/parsing/CliParsing.py
> <https://github.com/coala-analyzer/coala/pull/1863#discussion_r55932497>:
>
> > @@ -87,3 +87,31 @@ def parse_custom_settings(sections,
> >                                 origin=origin,
> >                                 section_name=key_touple[0],
> >                                 from_cli=True)
> > +
> > +
> > +def check_conflicts(cli_sections, log_printer):
>
> But it's not clear how to make [-a | -b] [-a | -c].
> mutually_exclusive_group only allows [-a | -b | -c]. I'll look into it
> more.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1863/files#r55932497>.
>